api/v1/ptpconfig_types.go

```diff
@@ -66,6 +66,20 @@ type PtpProfile struct {
 	Ptp4lOpts   *string `json:"ptp4lOpts,omitempty"`
 	Phc2sysOpts *string `json:"phc2sysOpts,omitempty"`
 	Ptp4lConf   *string `json:"ptp4lConf,omitempty"`
+	// +kubebuilder:default={}
```

Comment:
I think you can remove this it can be just nil

Reply:
having default allows me to populate with default values. Or users will not know the default values unless they look for logs.

Reply:
time="2021-07-19T19:13:33Z" level=info msg="update ptp threshold values for ens5f1\n holdoverTimeout: 5\n maxThreshold: 100\n minThreshold: -100\n"

Reply:
removed the default and checking for nil in sidecar